### PR TITLE
import fsp reconciliation AB#10289

### PR DESCRIPTION
**Pull request by jannisvisser**

### CODE REVIEW

File: services/121-service/src/registration/services/bulk-import.service.ts [hunk 1]
```diff
@@ -380,4 +389,100 @@ export class BulkImportService {
...
+
+      importResponseRecord.importStatus = ImportStatus.imported;
+      importResponseRecords.push(importResponseRecord);
+      countImported += 1;
+    }
+
+    this.actionService.saveAction(
+      userId,
+      programId,
+      AdditionalActionType.importPeopleAffected,
```
Comment (RubenGeo) [inline services/121-service/src/registration/services/bulk-import.service.ts:441]: `AdditionalActionType.importPeopleAffected` should be a new value?

File: services/121-service/src/registration/services/bulk-import.service.ts [hunk 2]
```diff
@@ -380,4 +389,100 @@ export class BulkImportService {
...
+        relations: ['fsp'],
+      });
+      if (!registration) {
+        importResponseRecord.importStatus = ImportStatus.unmatched;
+        importResponseRecords.push(importResponseRecord);
+        countNotFound += 1;
+        continue;
+      }
+
+      const paTransactionResult = new PaTransactionResultDto();
```
Comment (RubenGeo) [inline services/121-service/src/registration/services/bulk-import.service.ts:420]: What happens if you import the same file twice? You get multiple of the same transactions with later timestamps in the database?

Comment (jannisvisser) [reply services/121-service/src/registration/services/bulk-import.service.ts:420]: Yes, but that's no problem right? That's how our transactions system is set up, you can send as many status updates as you want, and always the latest one is taken. Right?

File: services/121-service/src/registration/services/bulk-import.service.ts (hunk 1 above)
Comment (jannisvisser) [reply services/121-service/src/registration/services/bulk-import.service.ts:441]: yes.

Comment (jannisvisser) [reply services/121-service/src/registration/services/bulk-import.service.ts]: done

File: services/121-service/src/registration/services/bulk-import.service.ts (hunk 2 above)
Comment (RubenGeo) [reply services/121-service/src/registration/services/bulk-import.service.ts:420]: But then the transaction date changes right. Do we not use that somewhere? Don't we want to differentiate between PAs that received their payment during an earlier import?

Comment (jannisvisser) [reply services/121-service/src/registration/services/bulk-import.service.ts:420]: Sure. I've made task to not update transaction if status stays the same.

**Comment by jannisvisser** [comment]
Closing this: see https://dev.azure.com/redcrossnl/121%20Platform/_sprints/taskboard/121%20Development%20Team/121%20Platform/Sprint%2064?workitem=10289

**Pull request closed without merging**